methods/calls/postCallsCallIdGather.md

```diff
@@ -15,7 +15,7 @@ Collects a series of DTMF digits from a phone call with an optional prompt. This
 |:-------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------|
 | maxDigits          | The maximum number of digits to collect, not including terminating digits (maximum 30).                                                                                                                                                                                                                                                                                                                                                                                             | No        |
 | interDigitTimeout  | Stop gathering if a DTMF digit is not detected in this many seconds (default 5.0; maximum 30.0).                                                                                                                                                                                                                                                                                                                                                                                    | No        |
-| terminatingDigits  | A string of DTMF digits that end the gather operation immediately if any one of them is detected (default “\#”; an empty string means collect all DTMF until maxDigits or the timeout).<br>Example:<br>`#` : The gather ends if \# is detected (this is the default behavior if the terminatingDigits property is not specified)<br>`0#*` : The gather ends if either 0, \#, or \* is detected<br>Don’t forget to encode keypad digits that have special meaning in a URL, like \#. | No        |
+| terminatingDigits  | A string of DTMF digits that end the gather operation immediately if any one of them is detected (default “\#”). Cannot be empty. <br>Example:<br>`#` : The gather ends if \# is detected (this is the default behavior if the terminatingDigits property is not specified)<br>`0#*` : The gather ends if either 0, \#, or \* is detected<br>Don’t forget to encode keypad digits that have special meaning in a URL, like \#. | No        |
```

Comment:
If it cannot be empty, should we set it to required? Or is it just that you can't supply it empty.

Recommend "If set, this cannot be empty" if not required, or simply changing the required field to yes and removing this sentence if it is.